setup-homebrew/action.yml

```diff
@@ -8,15 +8,15 @@ inputs:
   core:
     description: Update the `homebrew/core` tap.
     required: false
-    default: true
+    default: false
   cask:
     description: Update the `homebrew/cask` tap.
     required: false
-    default: true
+    default: false
   test-bot:
     description: Install the `homebrew/test-bot` tap.
     required: false
-    default: true
+    default: false
```

Comment:
This will be a breaking change. I recommend having the default be `auto` and that means true for taps and false otherwise (e.g. when `HOMEBREW_TAP_REPOSITORY` is set by the script).

Reply:
Yeh, that makes sense to me.

Reply:
@Bo98 honestly this is fast enough, though, I'm tempted to just keep it enabled.

Reply:
@Bo98 Should be fixed!